bedrock/firefox/tests/test_firefox_details.py

```diff
@@ -399,6 +399,58 @@ def test_latest_major_version_no_int(self):
         """latest_major_version should return 0 when no int."""
         eq_(firefox_desktop.latest_major_version('release'), 0)
 
+    @override_settings(FUNNELCAKE_LOCALES=('en-US'), FUNNELCAKE_PLATFORMS=('win'))
```

Comment:
unfortunately, due to how python deals with parentheticals, these are still just strings. In order for them to be tuples it'll have to be `('en-US',)`, or you could make them lists with `['en-US']`. It's confusing, but it's how the syntax works with parentheses. The tests may pass though, since `'win' in 'win'` is `True`, but it's not testing what we want.